modules/client-configuration/pages/vhm-caasp.adoc

```diff
@@ -57,3 +58,441 @@ When using Salt or {productname} (either via UI or API) on any {caasp} nodes:
+</scripts>
+</profile>
+----
+
+== Manage a {caasp} Cluster With {productname}
+
+You can use {productname} to manage one or more existing {caasp} clusters.
+
+[NOTE]
+====
+Only {caasp}{nbsp}4 is currently supported.
+====
+
+
+Before you begin, ensure you have installed your {caasp} cluster.
+You will need to manually copy some configuration information from your cluster to the {productname} Server:
+
+* Copy the ``skuba`` configuration directory from your cluster to the {productname} Server, and take a note of the file location.
+// Default file location? --LKB 2020-06-04
+This is the directory that the ``skuba`` service creates after the cluster has been bootstrapped.
+* Copy the passwordless private SSH key used to access the cluster nodes to the {productname} Server, and take a note of the file location.
+You need the current keys, and keys for any clients that you want to use in the future.
+Alternatively, you can use an ``ssh-agent`` socket, and provide the path to the socket when setting up the cluster.
+
+
+
+=== Elect a Management Node
+
+To manage a {caasp} cluster, you need to elect a client as the management node for the cluster.
+The management node cannot be part of the cluster, and it must have the {caasp} channels associated with it before you begin.
+You can use a single management node for multiple clusters, as long as the clusters are all of the same kind.
+
+
+
+.Procedure: Electing a Management Node
+. In the {productname} {webui}, navigate to menu:Systems[System List] and click the name of the client to elect as the management node.
+. Navigate to the menu:Formulas[Configuration] tab, and check the ``CaaSP Management Node`` formula.
+. Click btn:[Save] and apply the highstate.
+
+
+[NOTE]
+====
+You will not be able to use the management node until the highstate has been completed.
+====
+
+
+List all known clusters by navigating to menu:Clusters[Overview].
+This list displays all existing clusters, along with the cluster type, and which management node they are associated with.
+It also shows the nodes within the cluster, if the nodes are registered to {productname}.
+For the nodes within a cluster, additional information from ``skuba`` and the {k8s} API are shown, including the role, status, and whether any updates are available.
+
+For more information about the data available for nodes, see https://documentation.suse.com/suse-caasp/4/html/caasp-admin/_cluster_updates.html.
+
+
+
+=== Manage Clusters
+
+To manage a cluster in {productname}, add the cluster in the {webui}.
+
+
+
+.Procedure: Adding an Existing Cluster
+. In the {productname} {webui}, navigate to menu:Clusters[Overview] and click btn:[FIXME].
+. Follow the prompts to provide information about your cluster, including the cluster type, and select the management node to associate.
+. Type the path to the ``skuba`` configuration file for the cluster.
+// For example?
+. Type the passwordless SSH key you want to use, or to the ``ssh-agent`` socket.
```

Comment:
Maybe it would be useful to add a note here on how to forward the `ssh-agent`, something like:

There are two ways of using the ``ssh-agent`` with {caasp}:
1. Start the ``ssh-agent`` locally: ``eval $(ssh-agent)`` and add the ssh key: ``ssh-add <key>``. The socket used to access the agent is available in the ``$SSH_AGENT`` environment variable. 
2. Forward the ``ssh-agent`` to the management node from another machine: ``ssh -A <management node>``. The socket path is also available in the ``$SSH_AGENT`` environment variable. 

In both cases the path of the socket changes every time a new ``ssh-agent`` is started or a new ``ssh -A`` connection is started. 

The ``ssh-agent`` socket path can be updated at any time from the UI. The socket path can also be overridden when starting any cluster action that requires SSH access.

Reply:
Added